includes/api/class-wc-rest-product-categories-controller.php

```diff
@@ -62,11 +62,14 @@ public function prepare_item_for_response( $item, $request ) {
 				'date_created_gmt'  => wc_rest_prepare_date_response( $attachment->post_date_gmt ),
 				'date_modified'     => wc_rest_prepare_date_response( $attachment->post_modified ),
 				'date_modified_gmt' => wc_rest_prepare_date_response( $attachment->post_modified_gmt ),
-				'src'               => wp_get_attachment_url( $image_id ),
+				'src'               => wc_get_term_src_url( $item, $image_id, false, false ),
 				'name'              => get_the_title( $attachment ),
 				'alt'               => get_post_meta( $image_id, '_wp_attachment_image_alt', true ),
 			);
 		}
+		else{
```

Comment:
This does not follow coding standards - same mistake in multiple files.

Reply:
Can you point me which area doesn't follow standards? commas, separated with spaces, etc... ?

Reply:
See https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style

Reply:
I've updated my forked files you told me (Now braces uses the standards as it was said).